comment_scan/seclient.rb

```diff
@@ -0,0 +1,50 @@
+require 'se/api'
+
+class SEClient
+    def initialize(apiKey, site)
+        @site = site
+        @client = SE::API::Client.new(apiKey, site: site)
+    end
+
+    def post_exists?(post_id)
+        the_post = @client.posts(post_id.to_i)
+        if the_post.empty?
+            nil
+        else
+            the_post.first
+        end
+    end
+
+    def comment_deleted?(comment_id)
+        comment_with_id(comment_id) == nil
+    end
+
+    def comment_with_id(comment_id)
+        client_response = @client.comments(comment_id)
+        comment = client_response.empty? ? nil : client_response.first
```

Comment:
#first returns nil if the array is empty. Also, you don't need to assign to a variable, just return it.